Osiris/Config.cpp

```diff
@@ -1141,6 +1181,42 @@ void Config::reset() noexcept
     misc = { };
 }
 
+bool Config::fromClipboard() noexcept 
+{
+    char* clip;
+    if (OpenClipboard(NULL)){
+        clip = (char*)GetClipboardData(CF_TEXT);
```

Comment:
You should use `CF_UNICODETEXT` instead to support having unicode characters in config. Then use `WideCharToMultiByte()` function to convert from UTF-16 to a multibyte UTF-8 string.

Reply:
yeah i cant figure this one out, no matter what i try I seem to get random characters coming out from this. (`絻ਬ††絻ਬ††絻ਬ††絻ਬ††絻ਬ††絻ਬ††絻ਬ††絻ਬ††絻ਬ††絻ਬ††絻ਬ††絻ਬ††`)
If anyone else wants to pick this up, please do. I think this could be a useful feature